agents/Group23/mcts.py

```diff
@@ -150,4 +154,61 @@ def _default_policy(self, moves: list[Move]) -> Move:
         """
         if len(moves) == 0:
             raise ValueError("No legal moves available")
-        return random.choice(moves)
\ No newline at end of file
+        return random.choice(moves)
+    
+    def _removeInferiorCells(self, moves, board):
+
+        movesOut = []   
```

Comment:
Did a bit of reading and apparently list comprehension will run faster cuz of Python magic - maybe replace this section with: 
`moves = [move for move in moves if not self._patternMatch(move, board)]`
?